### ISSUE TITLE
Gamecrash on first CP in custom COTD

### ISSUE THREAD

**Issue by Freeman999** [thommie-echo/TMNext-PlayerState#11]
I got a gamecrash on the first CP in the first knockout round in a custom COTD. Also a few months ago same issue on the official nadeo COTD.
Someone also mentioned that having skindownloads activated or having too many plugins might cause an issue.

**Comment by thommie-echo** [comment]
Thanks for the report, would you be able to provide a list of plugins you're using and your log in case it crashes again?

**Comment by XertroV** [comment]
I get this via cotd delta time -- but not on custom COTD, on the nadeo ones. I usually play COTN and COTM (but not COTD due to TZs). I had been trying to debug the crash and after hard-disabling cotd delta time I haven't had the crash since. It was somewhat regular before (maybe 30% of COTDs, mb less) enough that twice in one day wasn't uncommon. I don't *know* that the crash was due to *cotd delta time* but it's the best lead I've had. (prior hypothesis was to do with setting a pb in no KO round.)

**Comment by XertroV** [comment]
> and your log in case it crashes again?

I checked my log many times and only once did I find anything useful (mostly it just ended abruptly with no useful info):

Usually there was nothing, but one crash where I reloaded the openplanet script engine (developer > restart script engine) there was a whole bunch of log msgs (once per frame?) from cotd delat time about not being able to find a player UID -- I think that was in the manialink stuff. (I assume the player was one of the players around the cutoff; my ID appeared sometimes I think but I was also at-risk during that match, from memory.)

I once confirmed that a plugin caused the crash by stopping the script engine for the no KO and 1st round, and then enabling it after. I got a crash on the 1st CP of the 2nd round.
